React/Views/RCTViewManager.m

```diff
@@ -82,6 +82,16 @@ - (RCTViewManagerUIBlock)uiBlockToAmendWithShadowViewRegistry:(RCTSparseArray *)
 RCT_REMAP_VIEW_PROPERTY(shadowOpacity, layer.shadowOpacity, CGFloat)
 RCT_REMAP_VIEW_PROPERTY(shadowRadius, layer.shadowRadius, CGFloat)
 RCT_REMAP_VIEW_PROPERTY(transformMatrix, layer.transform, CATransform3D)
+RCT_CUSTOM_VIEW_PROPERTY(rotation, CGFloat, RCTView)
+{
+  if (json) {
+    CGFloat angle = [RCTConvert CGFloat:json];
+    CGAffineTransform transform = CGAffineTransformRotate(view.transform, angle);
+    view.transform = transform;
+  } else {
+    view.transform = defaultView.transform;
+  }
```

Comment:
@fisherwebdev it might not be needed, I suppose. My understanding was that it would reset the `transform` in the case where `rotation: null` is found in the JSON, which I picked up from the convention of the other custom view properties. I could be wrong though, in which case I can remove it.